images/casperjs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN cd /
```

Comment:
These run commands are unlikely to change, right? Every single RUN adds a new layer to the image file, increasing its size. Use a single RUN with multiple chained `&&` commands to avoid that and keep the image tiny.

Reply:
item removed